src/mixins/itext_key_behavior.mixin.js

```diff
@@ -102,8 +102,8 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
       clipboardData.setData('text', selectedText);
     }
 
-    this.copiedText = selectedText;
-    this.copiedStyles = this.getSelectionStyles(
+    fabric.copiedText = selectedText;
+    fabric.copiedStyle = this.getSelectionStyles(
```

Comment:
Good idea! I would just call it `copiedTextStyle` so that it's clear what kind of style these are and in case we add something else in the future.